pkg/doc/did/doc.go

```diff
@@ -115,7 +115,7 @@ type PublicKey struct {
 
 	Value []byte
 
-	jsonWebKey *jose.JWK
+	JSONWebKey *jose.JWK `json:"-"`
```

Comment:
why we need it? why can't we simply figure out key type by "Type" field and parse value to JWK?